scripts/build/get_source/termux_git_clone_src.sh

```diff
@@ -17,6 +17,13 @@ termux_git_clone_src() {
 		git submodule update --init --recursive --depth=1
 		popd
 
+		ACTUAL_CHECKSUM=$(git -c core.abbrev=no -C "$TMP_CHECKOUT" archive --format tar "$TERMUX_PKG_GIT_BRANCH" | sha256sum 2>&1)
```

Comment:
This should be quoted.
```suggestion
		ACTUAL_CHECKSUM="$(git -c core.abbrev=no -C "$TMP_CHECKOUT" archive --format tar "$TERMUX_PKG_GIT_BRANCH" | sha256sum 2>&1)"
```